charts/kubeclarity/values.yaml

```diff
@@ -37,13 +37,16 @@ kubeclarity:
 
   enableDBInfoLog: false
 
-  podAnnotations: {}
+  # A value greater than 0 will enable the Prometheus metrics endpoint:
+  prometheusRefreshIntervalSeconds: 0
```

Comment:
I think it will be better to create section for `prometheus` (with an enable bit) if we will want in the future to add more values under

```
prometheus:
  enabled: false
  refreshIntervalSeconds: 300
```

I think it is more clear how to enable/disable it. WDYT?